avm/res/data-factory/factory/linked-service/main.bicep

```diff
@@ -41,6 +41,7 @@ resource linkedService 'Microsoft.DataFactory/factories/linkedservices@2018-06-0
           referenceName: integrationRuntimeName
           type: 'IntegrationRuntimeReference'
         }
+    #disable-next-line BCP225 // false-positive as 'type' is interpreted as a syntax value
```

Comment:
Please help me to understand why this line of code is here?

Reply:
This snippet suppresses a warning Bicep shows you in VSCode and when building the template because it thinks it cannot resolve the value in time (presumably because it interprets a value `type` as something else than a simple parameter/variable: 
![image](https://github.com/user-attachments/assets/f3e5ae84-f0a8-4f3f-b1d2-2466917c262a)

Just suppressing it here so that not every user of the module sees this warning they cannot do anything about.